.github/workflows/checks.yaml

```diff
@@ -42,4 +42,4 @@ jobs:
         godot --path . --headless --import
     - name: Run tests
       run: |
-        godot --path . --headless --script addons/gut/gut_cmdln.gd -gexit
+        godot --path . --headless -s addons/gut/gut_cmdln.gd -gexit
```

Comment:
This is also referenced in the [README](https://github.com/endlessm/godot-block-coding/blob/main/addons/block_code/README.md#testing). Might you update those instructions, too?

Reply:
Done – I also added a note that `--script` can't be used.